index.html

```diff
@@ -0,0 +1,180 @@
+                            <a href="#" class="linkB">OW Inno.</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="nav_wrapper">
+                    <nav class="desktop_nav">
+                        <div class="nav-logo">
+                            <a href="./index.html"><img src="images/logo-removebg-preview.png" class="nav-logo" alt="logo of Orbit Web Innovations"></a>
+                        </div>
+                        <ul class="nav_wrapper_desk">
+                            <li class="nav_items_desk">
+                                <a href="./about.html" class="link_deskA">About</a> 
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Program</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#courses" class="link_desk">Courses</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#lecturers" class="link_desk">Lecturers</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#partners" class="link_desk">Partners</a>
+                            </li>
+                            <li class="nav_items_desk">
+                                <a href="#" class="link_deskB">OW Inno.</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>
+    </header>
+    <section class="hero">
+                <div class="hero_box">
+                    <h1 class="hero_heading">"Hello! Sharing World"</h1>
+                    <h2 class="hero_title">ORBIT WEB INNOVATIONS</h2>
+                </div>
+                <div class="hero_description">
+                    <p class="des1">At Orbit institute we believe in nurturing great minds to become web programming professionals through our tailor-made courses. </p>
+                </div>
+                    <p class="des2">
+                        COURSE-TIME: (MON) ~ (FRI)
+                    </p>
+                    <p class="des3">
+                        @ Orbit Web Innovations. Oslo, Norway.
+                    </p>
+    </section>
+
+    <main id="courses" class="courses">
+        <div class="main">
+            <h2 class="programs">Main Courses</h2>
+            <hr class="hr-heading">
+        </div>
+        <div class="course_catalogue">
+            <ul class="python">
+                <li><img src="images/lecture-icon.png" class="icon" alt="lecture icon"></li>
+                <li class="python_course">Python</li>
+                <li class="python_des">A course in Python would teach you the basics of Python and how you can use it to do your basic coding.</li>
+            </ul>
```

Comment:
- [ ] Since the cards in this section are a self-contained entity on the page, I think it would be more appropriate to use the  semantic `article` tag instead of a `ul` tag. You can check this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) to get more insight on when to use the `article` element. ✍️

**Example**
```
<article class="python">
  <img src="images/lecture-icon.png" class="icon" alt="lecture icon">
  <h3 class="python_course">Python</h3>
  <p class="python_des">A course in Python would teach you the basics of Python and how you can use it to do your basic coding.</p>
</article>
```
This is just an example, you can tweak it to your needs. Please implement this structure on all the 5 articles in this section. 😎️